srcpkgs/unicorn/template

```diff
@@ -2,7 +2,7 @@
 pkgname=unicorn
 version=1.0.2
 revision=2
```

Comment:
This would need a revbump.

Reply:
As we discussed on IRC, it shouldn't be necessary because all remaining subpackages will still install the same files.